packages/dd-trace/src/config.js

```diff
@@ -823,7 +826,11 @@ class Config {
       this._envUnprocessed.peerServiceMapping = DD_TRACE_PEER_SERVICE_MAPPING
     }
     this._setString(env, 'port', DD_TRACE_AGENT_PORT)
-    const profilingEnabledEnv = coalesce(DD_EXPERIMENTAL_PROFILING_ENABLED, DD_PROFILING_ENABLED)
+    const profilingEnabledEnv = coalesce(
+      DD_EXPERIMENTAL_PROFILING_ENABLED,
+      DD_PROFILING_ENABLED,
+      this._isInServerlessEnvironment() ? 'false' : undefined
```

Comment:
Ooh, I see what you did here. Clever 😃.

For consistency, I'd probably move these to `_applyCalculated`. I know it's not _particularly_ interesting since you said we don't report telemetry but it's still conceptually cleaner. You could just put this in `_applyCalculated()`:
```javascript
    if (this._isInServerlessEnvironment()) {
      this._setValue(calc, 'profiling.enabled', 'false')
      this._setValue(calc, 'crashtracking.enabled', false)
    }
```

Reply:
Ironically, I moved them here _for consistency_, as other options are also calculated in this way in `_applyEnv`. For now let's keep it there and we can revisit in a separate PR if needed, although I kind of think it makes sense to be in `_applyEnv` as Serverless is an environment.

Reply:
I thought "Env" in "_applyEnv" meant "environment variables". But you're right that some uses of specifically `_isInServerlessEnvironment` have already snuck in, so by that reasoning I can accept these being there as well. If someone gets sufficiently irritated by it, they can fix it later.